examples/80_extending/example_extending_data_preprocessor.py

```diff
@@ -23,22 +23,20 @@ class NoPreprocessing(AutoSklearnPreprocessingAlgorithm):
 
     def __init__(self, **kwargs):
         """ This preprocessors does not change the data """
-        self.preprocessor = None
+        # Some internal checks makes sure parameters are set
```

Comment:
What do you mean by `some internal checks`?

Reply:
I'm not entirely sure where it was but there is a check somewhere in the pipeline that checks that certain attributes are set on the object from the `**kwargs` hence the snippet right below this
```python
for key, val in kwargs.items():
    setattr(self, key, val)
```